packages/examples/src/graphql/basic-query.ts

```diff
@@ -614,6 +614,71 @@ response = {
 
 console.log(response);
 
+/*
+ * Filter Patient.name by FHIRPath expression
+ */
+
+/*
+// start-block FilterExtensionByFHIRPathGraphQL
+{
+  PatientList {
+    resourceType
+    id
+    name(fhirpath: "use = 'anonymous' or (given.exists() and family.exists())") {
+      use given family
+    }
+  }
+}
+// end-block FilterExtensionByFHIRPathGraphQL
+*/
+
+// start-block FilterPatientNameByFHIRPathTS
+await medplum.graphql(`{
+  PatientList {
+    resourceType
+    id
+    name(fhirpath: "use = 'anonymous' or (given.exists() and family.exists())") {
```

Comment:
This query doesn't work at the moment 😬 

<img width="875" alt="image" src="https://github.com/medplum/medplum/assets/749094/b04b0dbb-6bbb-4683-8bb3-140554c54e4d">